packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx

```diff
@@ -115,6 +116,14 @@ export const PayablesTable = (props: PayablesTableProps) => (
   </MoniteScopedProviders>
 );
 
+const usePayablesTableSummaryData = () => {
```

Comment:
Shouldn't this hook be placed in the hooks folder?

Reply:
Discussed directly, no longer issue